.github/workflows/linux_meson.yml

```diff
@@ -278,7 +278,7 @@ jobs:
 
         # Install OpenBLAS a la cibuildwheel.
         chmod +x tools/wheels/cibw_before_build_linux.sh
-        sudo tools/wheels/cibw_before_build_linux.sh .        
+        sudo tools/wheels/cibw_before_build_linux.sh --nightly .       
```

Comment:
If we're starting to create command line options to a bash script is it worth using  [`getopt`](https://www.baeldung.com/linux/bash-parse-command-line-arguments)? If `--nightly` and `.` are swapped around then the script won't work as intended.